app/Http/Controllers/RedirectsController.php

```diff
@@ -42,7 +42,17 @@ public function create()
      */
     public function store(Request $request)
     {
-        $redirect = Redirect::create($request->all());
+        $this->validate($request, [
+            'path' => 'required',
+            'target' => 'required',
+        ]);
+
+        $request = $request->all();
+        if(substr($request['path'], 0, 1) !== '/')
+        {
```

Comment:
This bracket should be on the same line as the `if`, and no-space between `if` and `(...)`! :space_invader: